templates/basic/package.json

```diff
@@ -6,6 +6,8 @@
     "prepare": "nuxt prepare"
   },
   "devDependencies": {
+    "@nuxtjs/color-mode": "^3.3.2",
+    "@unocss/nuxt": "^0.57.7",
```

Comment:
I wonder if there is a way to avoid introducing them, because:

- We want the example to be simple and not introduce too much concept
- We want the container to boot as fast as possible, so it would be better to avoid introduce more dependencies

Reply:
I agree, this is not a good approach in terms of performance. I will try to come up with something with passed inline styles